src/state/is-within-visible-bounds-of-droppable.js

```diff
@@ -9,7 +9,12 @@ import type {
   Spacing,
 } from '../types';
 
-export const isPointWithin = (droppable: DroppableDimension) => {
+const noPadding: Position = { x: 0, y: 0 };
+
+const getVisibleBounds = (
```

Comment:
rather than adding an additional parameter here, perhaps we should create a function that add a `Position` or a `Spacing` to a `DroppableDimension`